appmesh-preview/sdk/api.normal.json

```diff
@@ -3282,12 +3287,15 @@
       "type": "structure",
       "members": {
         "idle": {
-          "shape": "Duration"
+          "shape": "Duration",
+          "documentation": "<p>An object that represents an idle timeout. An idle timeout bounds the amount of time that a connection may be idle. The default value is none.</p>"
         },
         "perRequest": {
-          "shape": "Duration"
+          "shape": "Duration",
+          "documentation": "<p>An object that represents a per request timeout. The default value is 15 seconds. If you set a higher timeout, then make sure that the higher value is set for each App Mesh resource in a conversation. For example, if a virtual node backend uses a virtual router provider to route to another virtual node, then the timeout should be greater than 15 seconds for the source and destination virtual node and the route.</p>"
```

Comment:
The "for the source and destination" could use some clarification. Shouldn't it be the other way around? This is what I get from the description: 

`[VN w/ backend timeout=20] --> [VS] --> [VR + Route] --> [VN timeout=25]`

But logically it seems like the VN using the other service as a backend should have a larger timeout. Like this:

`[VN w/ backend timeout=25] --> [VS] --> [VR + Route] --> [VN timeout=20]`

Reply:
I think this is auto-generated from our doc-model, can take this discussion offline.